lib/node/api.js

```diff
@@ -971,6 +972,33 @@ api.getFunds = async (req, res) => {
   }
 };
 
+const handleSingleTxDetail = async (txDetailParam, address) => {
+  let totalInputValue = 0;
+  let totalOutputValue = 0;
+  const txDetail = txDetailParam;
+  txDetail.transactionType = omniTxTypes.incoming;
+  const inputTxDetailPromises = txDetailParam.vin.map(async (vin) => {
```

Comment:
Can we remove the `async` keyword here too?

Reply:
Fixed here: https://github.com/imthe-1/multi-dimension-identity-protocol/pull/4/files/c2c09fb7cdf6ce9928e8381e61347c6d30d69f3c..f96fa1ac8fedb5403059bd5121c9a430c3ae36c2